doc/database/db_post-media.md

```diff
@@ -6,27 +6,27 @@ Attached media
-| type            | Media type                                                | tinyint unsigned   | NO   |     | 0       |                |
-| mimetype        |                                                           | varchar(60)        | YES  |     | NULL    |                |
-| height          | Height of the media                                       | smallint unsigned  | YES  |     | NULL    |                |
-| width           | Width of the media                                        | smallint unsigned  | YES  |     | NULL    |                |
-| size            | Media size                                                | mediumint unsigned | YES  |     | NULL    |                |
-| preview         | Preview URL                                               | varbinary(255)     | YES  |     | NULL    |                |
-| preview-height  | Height of the preview picture                             | smallint unsigned  | YES  |     | NULL    |                |
-| preview-width   | Width of the preview picture                              | smallint unsigned  | YES  |     | NULL    |                |
-| description     |                                                           | text               | YES  |     | NULL    |                |
-| name            | Name of the media                                         | varchar(255)       | YES  |     | NULL    |                |
-| author-url      | URL of the author of the media                            | varbinary(255)     | YES  |     | NULL    |                |
-| author-name     | Name of the author of the media                           | varchar(255)       | YES  |     | NULL    |                |
-| author-image    | Image of the author of the media                          | varbinary(255)     | YES  |     | NULL    |                |
-| publisher-url   | URL of the publisher of the media                         | varbinary(255)     | YES  |     | NULL    |                |
-| publisher-name  | Name of the publisher of the media                        | varchar(255)       | YES  |     | NULL    |                |
-| publisher-image | Image of the publisher of the media                       | varbinary(255)     | YES  |     | NULL    |                |
+| Field           | Description                                               | Type              | Null | Key | Default | Extra          |
+| --------------- | --------------------------------------------------------- | ----------------- | ---- | --- | ------- | -------------- |
+| id              | sequential ID                                             | int unsigned      | NO   | PRI | NULL    | auto_increment |
+| uri-id          | Id of the item-uri table entry that contains the item uri | int unsigned      | NO   |     | NULL    |                |
+| url             | Media URL                                                 | text              | NO   |     | NULL    |                |
```

Comment:
The `url` field mustn't be a `text` since we use it in an unique index-

Reply:
Then we will have again the same error about being to short for super-long URLs like `magnet:` URLs can be. That's why I did these both changes, to cover long URLs and big video files. If I change both back to original vales we have the same two DB errors back.